ui/src/composables/setupIMQueryBuilderActions.ts

```diff
@@ -0,0 +1,87 @@
+import { isArrayHasLength, isObjectHasKeys } from "@im-library/helpers/DataTypeCheckers";
+import { Bool, Match, Query, Where } from "@im-library/interfaces/AutoGen";
+import { MenuItem } from "primevue/menuitem";
+
+function setupIMQueryBuilderActions() {
+  function isPathMatch(match: Match): boolean {
+    return (match.typeOf && match.where && !match.match) as boolean;
```

Comment:
why casting?

Reply:
typescript workaround. return is seen as 'boolean | undefined' otherwise